go.mod

```diff
@@ -41,6 +41,7 @@ require (
 	github.com/filecoin-project/go-commp-utils v0.1.3
 	github.com/filecoin-project/go-commp-utils/nonffi v0.0.0-20220905160352-62059082a837
 	github.com/filecoin-project/go-crypto v0.0.1
+	github.com/filecoin-project/go-f3 v0.0.0-20240624015334-ee517df37076
```

Comment:
This commit does not seem to be on `main`.

Ideally we should depend on a concrete release.
At least we should not depend on a commit that is not on `main`.

Reply:
Yes, it is my latest PR, I'll update: https://github.com/filecoin-project/go-f3/pull/367

Reply:
We are now pointing to main